### DISCUSSION TITLE
how do i make the number line take up less space

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [zed-industries/zed#18736] [Feature Requests]
![image](https://github.com/user-attachments/assets/dc71eb75-a4dc-4bff-842a-97bbbf12ead8)

also how do i run the code without using the terminal. most ides have some sort of run code currently being displayed button. processing, visual studio, and thonny all have a button you can click or a command or button which just runs the code on screen

**Comment by thataboy** [comment]
Unfortunately you can't.

You can turn some of this off if you don't need them

```
  "gutter": {
    // Whether to show line numbers in the gutter.
    "line_numbers": true,
    // Whether to show code action buttons in the gutter.
    "code_actions": true,
    // Whether to show runnables buttons in the gutter.
    "runnables": true,
    // Whether to show fold buttons in the gutter.
    "folds": true
  },
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
